### PR TITLE
feat(060): Canonical Status Model Cleanup — 3.0 hard cutover

### PR DESCRIPTION
**Pull request by robertDouglass**
## Summary

- **Hard cutover**: `status.events.jsonl` is now the sole authority for WP lane state everywhere in the spec-kitty repo
- **Frontmatter lane removed** from active WP schema, templates, body-note writers, and runtime fallbacks
- **finalize-tasks** seeds canonical `planned` events for all WPs (both entrypoints)
- **Runtime hard-fail** when canonical state is missing — no silent frontmatter fallback
- **600+ regression guard tests** prevent reintroduction of frontmatter-lane patterns

### Phase A — Canonical Bootstrap
| File | Change |
|------|--------|
| `status/bootstrap.py` | **NEW** — shared `bootstrap_canonical_state()` helper |
| `feature.py` | finalize-tasks calls bootstrap + `--validate-only` + lane-free README generator |
| `tasks.py` | finalize-tasks calls bootstrap + `--validate-only`, lane= removed from body notes, bootstrap/sync block deleted |

### Phase B — Template + Body-Note Cleanup
| File | Change |
|------|--------|
| 10 mission/doctrine templates | `lane:` removed from frontmatter, `lane=` from activity logs, `history[].lane` from examples |
| 2 shared packaged templates | Same cleanup |
| `workflow.py` | `lane=` removed from implement/review body notes |
| `scripts/tasks/tasks_cli.py` | `lane=` removed from body logs, frontmatter lane write removed |

### Phase C — Runtime Fallback Removal
| File | Change |
|------|--------|
| `tasks_support.py` | `WorkPackage.lane` — event-log-only, no frontmatter fallback |
| `dashboard/scanner.py` | Both fallback branches removed, function renamed |
| `mission_v1/guards.py` | `_read_lane_from_frontmatter` redirected to lane_reader |
| `next/runtime_bridge.py` | Uses `get_wp_lane()` instead of frontmatter |
| `merge.py` | Frontmatter fallback removed |
| `workflow.py` | Implement/review frontmatter fallbacks removed, hard-fail added |
| `status/lane_reader.py` | `CanonicalStatusNotFoundError`, `has_event_log()`, "uninitialized" for missing WPs |

### Phase D — Migration Fence + Docs + Guards
| File | Change |
|------|--------|
| `task_metadata_validation.py` | `repair_lane_mismatch` marked MIGRATION-ONLY |
| `status/history_parser.py` | Marked MIGRATION-ONLY |
| `docs/status-model.md`, `CLAUDE.md` | Updated for 3.0 canonical model |
| `validate_tasks.py` | Docstring updated to legacy/migration-only |

## Test plan

- [x] Bootstrap: 15 tests (seed, skip, dry-run, edge cases)
- [x] feature.py integration: 4 tests
- [x] tasks.py changes: 6 tests (bootstrap, body notes, hard-fail)
- [x] workflow.py changes: 9 tests (body notes, hard-fail)
- [x] Template guard: 152 tests (scan all active templates for lane:)
- [x] Runtime hard-fail: 22 tests (event log absent, WP missing, normal)
- [x] Regression guard: 353 tests (scan all non-migration Python + scripts for frontmatter lane access)
- [x] Existing tests updated: conftest fixtures, 6 test files cleaned

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: src/specify_cli/cli/commands/implement.py [hunk 1]
```diff
@@ -1056,14 +1058,14 @@
                 )
 
             # Write updated WP file (always, regardless of auto_commit)
-            wp.path.write_text(updated_doc, encoding="utf-8")
+            wp_file.write_text(updated_doc, encoding="utf-8")
```
[the remark this answers is not in the document]
Comment (samuelgoff) [reply src/specify_cli/cli/commands/implement.py]: Audit 2026-09-05: the cited user-controlled filename construction is no longer live. Current find_wp_file validates the work-package identifier against the WP format and resolves the task directory through the canonical Mission placement seam before matching filenames. This security finding is addressed; thread resolution is blocked by repository permission.
